src/ArduinoJson/MsgPack/MsgPackBinary.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+ARDUINOJSON_BEGIN_PUBLIC_NAMESPACE
+
+class MsgPackBinary {
+ public:
+  explicit MsgPackBinary(const void* c, size_t size) : data_(c), size_(size) {}
+  operator const void*() const {
```

Comment:
I don't think the implicit conversion to `const void*` is a good idea.
Please remove it.